docs/source/user_guide/dataset_zoo/datasets.rst

```diff
@@ -3391,6 +3393,233 @@ annotation data.
+        # Load segmentations for 25 samples from the validation split that
+        # contain cats and dogs
+        #
+        # Images that contain all `classes` will be prioritized first, followed
+        # by images that contain at least one of the required `classes`. If
+        # there are not enough images matching `classes` in the split to meet
+        # `max_samples`, only the available images will be loaded.
+        #
+        # Images will only be downloaded if necessary
+        #
+
+        dataset = foz.load_zoo_dataset(
+            "sama-coco",
+            split="validation",
+            label_types=["segmentations"],
+            classes=["cat", "dog"],
+            max_samples=25,
+        )
+
+        session.dataset = dataset
+
+        #
+        # Download the entire validation split and load both detections and
+        # segmentations
+        #
+        # Subsequent partial loads of the validation split will never require
+        # downloading any images
+        #
+
+        dataset = foz.load_zoo_dataset(
+            "sama-coco",
+            split="validation",
+            label_types=["detections", "segmentations"],
+        )
+
+        session.dataset = dataset
+
+  .. group-tab:: CLI
+
+    .. code-block:: shell
+
+        #
+        # Load 50 random samples from the validation split
+        #
+        # Only the required images will be downloaded (if necessary).
+        # By default, only detections are loaded
+        #
+
+        fiftyone zoo datasets load sama-coco \
+            --split validation \
+            --kwargs \
+                max_samples=50
+
+        fiftyone app launch sama-coco-validation-50
+
+        #
+        # Load segmentations for 25 samples from the validation split that
+        # contain cats and dogs
+        #
+        # Images that contain all `classes` will be prioritized first, followed
+        # by images that contain at least one of the required `classes`. If
+        # there are not enough images matching `classes` in the split to meet
+        # `max_samples`, only the available images will be loaded.
+        #
+        # Images will only be downloaded if necessary
+        #
+
+        fiftyone zoo datasets load sama-coco \
+            --split validation \
+            --kwargs \
+                label_types=segmentations \
+                classes=cat,dog \
+                max_samples=25
+
+        fiftyone app launch sama-coco-validation-25
+
+        #
+        # Download the entire validation split and load both detections and
+        # segmentations
+        #
+        # Subsequent partial loads of the validation split will never require
+        # downloading any images
+        #
+
+        fiftyone zoo datasets load sama-coco \
+            --split validation \
+            --kwargs \
+                label_types=detections,segmentations
+
+        fiftyone app launch sama-coco-validation
+
+.. image:: /images/dataset_zoo/sama-coco-validation.png
```

Comment:
Include this image in the PR

Reply:
Image added - any requirements on aspect ratios?

Reply:
Thanks! No hard requirements on aspect ratios, the image you added looks great!